tests/Authentication/Authenticators/AccessTokenAuthenticatorTest.php

```diff
@@ -28,7 +28,7 @@ protected function setUp(): void
 
         $config = new Auth();
         $auth   = new Authentication($config);
-        $auth->setProvider(model(UserModel::class)); // @phpstan-ignore-line
+        $auth->setProvider(\model(UserModel::class)); // @phpstan-ignore-line
```

Comment:
Our style guide decided against slashing root functions. Did you add these, or did they slip in via CS Fixer/Rector somehow?

Reply:
It seems PhpStorm did it when refactoring.

Reply:
Removed.